packages/hdwallet-native/src/native.ts

```diff
@@ -55,12 +94,16 @@ class NativeHDWalletInfo
       case "testnet":
         const unknown = core.unknownUTXOPath(msg.path, msg.coin, msg.scriptType);
 
-        if (!super.btcSupportsCoin(msg.coin)) return unknown;
-        if (!super.btcSupportsScriptType(msg.coin, msg.scriptType)) return unknown;
+        if (!this.btcSupportsCoin(msg.coin)) return unknown;
+        if (!this.btcSupportsScriptType(msg.coin, msg.scriptType)) return unknown;
```

Comment:
Can this stay as super to show we are calling a function from an extended class.

Reply:
I read up on this before the change, and unless overridden, they are basically the same thing. Since it's the child, `this` works just fine. It's not really worth arguing about though.

Reply:
Sure, I would just say we are using super to call all parent class function except for this function now is all. I will leave it at that.